spec/services/fixed_region_info_spec.rb

```diff
@@ -0,0 +1,90 @@
+require "spec_helper"
+
+describe FixedRegionInfo do
+  it "calculates average, number of contributors, top contributors and number of samples for sessions for given stearm" do
+    session1 = create_session_with_stearms_and_measurements!({ id: 1, longitude: 1.0, latitude: 1.0, value: 1 })
+    session2 = create_session_with_stearms_and_measurements!({ id: 2, longitude: 1.0, latitude: 1.0, value: 2 })
+
+    data = {
+      session_ids: [session1.id, session2.id],
+      sensor_name: "AirBeam2-F",
+    }
+
+    region_info = FixedRegionInfo.new.call(data)
+
+    expect(region_info).to eq({
+        average: 1.5,
+        number_of_contributors: 2,
+        top_contributors: (["Test User1", "Test User2"]),
```

Comment:
It's difficult to understand why the users are called that way. You could make that explicit by creating the users in the test and having `username1` and `username2` in common